flash.wdl

```diff
@@ -0,0 +1,42 @@
+task flash {
+    String? preCommand
+    File inputR1
+    File inputR2
+    String outdirPath
+    String? outPrefix = "flash"
+    String? logPath = outdirPath + "/" + outPrefix + ".log"
+    Int? minOverlap
+    Int? maxOverlap
+    Boolean? compress = true
+    Int? threads = 1
+    Int? memory = 4
```

Comment:
The user should be able to change this, the default should be removed here. Related to comments below.